extensions/notification/NotificationConfigManager.js

```diff
@@ -418,7 +418,10 @@ class NotificationConfigManager {
      * @return {undefined}
      */
     init(cb) {
-        return this._checkConfigurationParentNode(() => {
+        return this._checkConfigurationParentNode(err => {
+            if (err) {
+                return cb(err);
+            }
```

Comment:
can this be tested? Same with the other.

Reply:
I was thinking about it, the error was uncaught in the original case (the bug) and that was the reason for the crash.

Reply:
It should be possible to add a test that causes an error and assert the error is called in the callback. That is usually how I test error checks like this.

Reply:
Yep, already made the change, pushing a commit.